src/protocols/ethereum/views/TransferSendModal.vue

```diff
@@ -62,9 +59,9 @@ export default defineComponent({
       address: props.address as any, // TODO change to string globally
       amount: props.amount,
       payload: props.payload,
-      selectedAsset: ProtocolAdapterFactory
-        .getAdapter(PROTOCOLS.ethereum)
-        .getDefaultCoin(marketData.value!, +balance.value),
+      selectedAsset: (props.tokenContractId)
+        ? availableTokens.value[props.tokenContractId]
+        : undefined,
```

Comment:
Are you sure `undefined` is OK here? Maybe the ETH coin should be used instead?

Reply:
This is based on this line https://github.com/superhero-com/superhero-wallet/blob/feat/eth-support/src/protocols/aeternity/views/TransferSendModal.vue#L77C2-L77C2
And IIRC `availableTokens` always has the default coin